cJSON.h

```diff
@@ -81,84 +81,113 @@ typedef struct cJSON_Hooks
       void (*free_fn)(void *ptr);
 } cJSON_Hooks;
 
+#if !defined(__WINDOWS__) && (defined(WIN32) || defined(WIN64) || defined(_MSC_VER) || defined(_WIN32))
+#define __WINDOWS__
+#endif
+#ifdef __WINDOWS__
+#if defined(CJSON_DECLARE_STATIC)
+#define CJSON_DECLARE(type)			type __stdcall
```

Comment:
If it is not possible to do this on old compilers otherwise, than keep it as it is.